caribou/deployment_solver/deployment_input/components/calculators/cost_calculator.py

```
@@ -165,16 +176,25 @@ def _get_execution_conversion_ratio(self, instance_name: str, region_name: str)
         # Get the number of vCPUs and Memory of the instance
         provider, _ = region_name.split(":")
         memory_mb: float = self._workflow_loader.get_memory(instance_name, provider)
+        vcpu: float = self._workflow_loader.get_vcpu(instance_name, provider)
 
         ## datacenter loader data
         architecture: str = self._workflow_loader.get_architecture(instance_name, provider)
-        compute_cost: float = self._datacenter_loader.get_compute_cost(region_name, architecture)
+        compute_cost: Any = self._datacenter_loader.get_compute_cost(region_name, architecture)
         invocation_cost: float = self._datacenter_loader.get_invocation_cost(region_name, architecture)
 
-        # Compute cost in USD / GB-seconds
-        # Memory in MB, execution_time in seconds, vcpu in vcpu
-        memory_gb: float = memory_mb / 1024
-        cost_from_compute_s: float = compute_cost * memory_gb  # IN USD / s
+        if provider == Provider.GCP.value:
+            # GCP cost in USD / GB-seconds + vCPU-seconds.
+            # Memory in MB, execution_time in seconds, vcpu in vcpu
+            memory_gb: float = memory_mb / 1024
+            memory_cost: float = compute_cost.get("memory_gb_s", 0)
+            vcpu_cost: float = compute_cost.get("cpu_s", 0)
```

Not quite sure about this default to zero, as it may potentially lead to regions with no data instead being cheaper than regions with data. Should we instead throw error if not available?

---

I see that datacenter loader have a default cost, which is great, so perhaps just remove the default to zero here and do no default values (and let it crash) if the keys doesn't exist.

---

Fixed, removed default value of 0